deployments/serializers.py

```diff
@@ -138,9 +138,9 @@ class Meta:
 
     @staticmethod
     def get_groups(obj):
-        return [t.name for t in obj.groups.all() if not t.is_deprecated]
+        return [t.name for t in obj.groups.filter(is_deprecated=False)]
```

Comment:
You're right, @batpad, in the [final version](https://github.com/IFRCGo/go-api/commit/dec56112e3e8cc49e005eb9caa70cfb554109393) I reverted the original one (it gave better results, because the dropping of some data on serializer level was cheaper then introduce a new SQL condition).